Description: Changes to introducing dpkg-buildflags
 This patch assists in introducing dpkg-buildflags in the rules
 file.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Last-Updated: 2012-02-16
Forwaded: not-needed
--- a/config.mk
+++ b/config.mk
@@ -19,9 +19,9 @@
 #       -ljavascriptcoregtk-1.0
 
 # flags
-CPPFLAGS = -DVERSION=\"${VERSION}\"
-CFLAGS = -std=c99 -pedantic -Wall -Os ${INCS} ${CPPFLAGS}
-LDFLAGS = -g ${LIBS}
+CPPFLAGS += -DVERSION=\"${VERSION}\"
+CFLAGS += -std=c99 -pedantic -Wall ${INCS} ${CPPFLAGS}
+LDFLAGS += -g ${LIBS}
 
 # Solaris
 #CFLAGS = -fast ${INCS} -DVERSION=\"${VERSION}\"
